### DISCUSSION TITLE
[BUG] cannot get the ROFI_DATA

### DISCUSSION THREAD

**Discussion by sunn4room** [davatorium/rofi#1626] [General]
### Rofi version (rofi -v)

1.7.3

### Configuration

no configuration

### Launch command

rofi -modi "example:./Examples/test_script_env.sh" -show example

### Step to reproduce

no step

### Expected behavior

no

### Actual behavior

no

### Additional information

my enviroment:
- system: manjaro i3
- linux: 5.15.32
- shell: zsh

**Comment by DaveDavenport** [comment]
No information provided, just replied no everywhere.   The example script works fine for me in git (just tested), but if you cannot be bothered to actually try to explain, I cannot fix it.

**Reply by DaveDavenport** [reply]
Moving this to where it is more appropriate and to answer for other people trying to solve this:
```
ROFI_DATA
``` 
is not a feature present in 1.7.3.  It is also not part of test scrip in 1.7.3:

```bash
 #!/usr/bin/env bash
 
 if [ -z "${ROFI_OUTSIDE}" ]
 then
     echo "run this script in rofi".
     exit
 fi
 
 echo -en "\x00no-custom\x1ffalse\n"
 echo -en "\x00use-hot-keys\x1ftrue\n"
 echo -en "${ROFI_RETV}\x00icon\x1ffirefox\x1finfo\x1ftest\n"
 
 if [ -n "${ROFI_INFO}" ]
 then
     echo "my info: ${ROFI_INFO} "
 fi
```
